cli/README.md

```diff
@@ -21,16 +21,49 @@ To run a computation, use the following command:
 ```
 
 #### Get attestation
+Retrieves attestation information from the SEV guest and saves it to a file.
 To retrieve attestation from agent, use the following command:
 ```bash
 ./build/cocos-cli agent attestation get '<report_data>'
 ```
 
 #### Validate attestation
+Validates the retrieved attestation information against a specified policy and checks its authenticity.
 To validate and verify attestation from agent, use the following command:
 ```bash
-./build/cocos-cli agent attestation validate '<attestation>' '<report_data>'
+./build/cocos-cli agent attestation validate '<attestation>' --report_data '<report_data>'
 ```
+##### Flags
```

Comment:
Can we add an option for `guest_policy`? Like here:

https://github.com/google/go-sev-guest/blob/main/tools/check/check.go

Reply:
we do have it as `guest_svn` https://github.com/ultravioletrs/cocos/blob/b0a81d400ca9a235fdf5f837a61df7af10c3a4c0/cli/attestation.go#L174

Reply:
Ok, I got confused for a moment because there is a  field in the attestation report called Guest SVN, which is not guest policy and the value for guest policy should initialize `cfg.Policy.Policy`.

Reply:
Is this resolved (`guest_policy` field is not added as far as I can see).